From a3feed56054bbebcdaedf89a9e7cdea5067cd6d0 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Wed, 20 Sep 2017 16:13:32 +0200
Subject: [PATCH] playback.cpp: avoid redefinition of uint64_t
 cPlayback::GetReadCount()

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 generic-pc/playback.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/generic-pc/playback.cpp b/generic-pc/playback.cpp
index 86b6b6f..f448db7 100644
--- a/generic-pc/playback.cpp
+++ b/generic-pc/playback.cpp
@@ -146,7 +146,3 @@ cPlayback::~cPlayback()
 	printf("%s:%s\n", FILENAME, __func__);
 }
 
-uint64_t cPlayback::GetReadCount()
-{
-	return 0;
-}
-- 
2.13.3

